frontend/src/components/common/Filter/common/constants.ts

```diff
@@ -1,4 +1,1121 @@
+      ontology_term_id: "UBERON:0002240",
+    },
+    {
+      label: "spleen",
+      ontology_term_id: "UBERON:0002106",
+    },
+    {
+      label: "stomach",
+      ontology_term_id: "UBERON:0000945",
+    },
+    {
+      label: "thymus",
+      ontology_term_id: "UBERON:0002370",
+    },
+    {
+      label: "thyroid gland",
+      ontology_term_id: "UBERON:0002046",
+    },
+    {
+      label: "tongue",
+      ontology_term_id: "UBERON:0001723",
+    },
+    {
+      label: "uterus",
+      ontology_term_id: "UBERON:0000995",
+    },
+  ],
+};
+/* eslint-enable sonarjs/no-duplicate-string -- disable no dupes, this value is hand-curated */
+/* eslint-enable sort-keys -- disabling key order for readability. */
+
+/**
+ * Tissues to be included for display in tissue system ontology tree.
+ */
+/* eslint-disable sort-keys -- disabling key order for readability. */
+export const TISSUE_SYSTEM_ONTOLOGY_TERM_SET: OntologyTermSet = {
+  [ONTOLOGY_VIEW_KEY.UBERON]: [
+    {
+      label: "central nervous system",
+      ontology_term_id: "UBERON:0001017",
+    },
+    {
+      label: "cardiovascular system",
+      ontology_term_id: "UBERON:0004535",
+    },
+    {
+      label: "circulatory system",
+      ontology_term_id: "UBERON:0001009",
+    },
+    {
+      label: "digestive system",
+      ontology_term_id: "UBERON:0001007",
+    },
+    {
+      label: "embryo",
+      ontology_term_id: "UBERON:0000922",
+    },
+    {
+      label: "endocrine system",
+      ontology_term_id: "UBERON:0000949",
+    },
+    {
+      label: "exocrine system",
+      ontology_term_id: "UBERON:0002330",
+    },
+    {
+      label: "hematopoietic system",
+      ontology_term_id: "UBERON:0002390",
+    },
+    {
+      label: "immune system",
+      ontology_term_id: "UBERON:0002405",
+    },
+    {
+      label: "musculature of body",
+      ontology_term_id: "UBERON:0000383",
+    },
+    {
+      label: "nervous system",
+      ontology_term_id: "UBERON:0001016",
+    },
+    {
+      label: "peripheral nervous system",
+      ontology_term_id: "UBERON:0000010",
+    },
+    {
+      label: "renal system",
+      ontology_term_id: "UBERON:0001008",
+    },
+    {
+      label: "reproductive system",
+      ontology_term_id: "UBERON:0000990",
+    },
+    {
+      label: "respiratory system",
+      ontology_term_id: "UBERON:0001004",
+    },
+    {
+      label: "sensory system",
+      ontology_term_id: "UBERON:0001032",
+    },
+    {
+      label: "skeletal system",
+      ontology_term_id: "UBERON:0001434",
+    },
+  ],
+};
+/* eslint-enable sort-keys -- disabling key order for readability. */
+
+/**
+ * Generic tooltip displayed when select, ontology or range category is disabled due to no values matching current
+ * filter.
+ */
+export const TOOLTIP_CATEGORY_DISABLED =
+  "There are no values that meet the current filter criteria";
+
+/**
+ * Configuration of each category filter.
+ */
+const CATEGORY_FILTER_CONFIGS: CategoryFilterConfig[] = [
```

Comment:
* Added tissue multi-panel configuration
* Added discriminating unions for all category filter types to help determine functionality switches for each.